.env.example

```
@@ -1,4 +1,6 @@
 SUPABASE_URL=<YOUR_SUPABASE_URL>
 SUPABASE_ANON_KEY=<YOUR_SUPABASE_ANON_KEY>
+SUPABASE_SERVICE_ROLE_KEY=<YOUR_SUPABASE_SERVICE_ROLE_KEY>
 GEMINI_API_KEY=<YOUR_GEMINI_API_KEY>
-    
\ No newline at end of file
+VEXA_API_KEY=<YOUR_VEXA_API_KEY>
+OPEAI_API_KEY=<YOUR_OPEAI_API_KEY>
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The OPEAI_API_KEY key should go before the SUPABASE_ANON_KEY key

(UnorderedKey)